### ISSUE TITLE
how kuma userd under kubernets

### ISSUE THREAD

**Issue by bruse-peng** [Kong/kuma#240]
### Summary

SUMMARY_GOES_HERE

### Steps To Reproduce

1.
2.
3.
4.

### Additional Details & Logs

- Version
- Error logs
- Configuration
- Platform and Operating System

can offer the functions kuma mange kubernets application,and give some examples

**Comment by subnetmarco** [comment]
Sorry, can you rephrase the question? If you are looking for instructions of Kuma on Kubernetes you can find them on https://kuma.io/install

**Comment by bruse-peng** [comment]
how  kuma used on kubernets can you offer some example

**Comment by subnetmarco** [comment]
You can find an example of Kuma on Kubernetes at https://kuma.io/docs/0.1.2/installation/kubernetes/
